src/Raven.Server/Documents/Queries/Results/QueryResultRetrieverBase.cs

```diff
@@ -729,7 +729,7 @@ private object InvokeFunction(string methodName, Query query, string documentId,
                 query.DeclaredFunctions.TryGetValue(methodName, out var func) &&
                 func.Type == DeclaredFunction.FunctionType.TimeSeries)
             {
-                _timeSeriesRetriever ??= new TimeSeriesRetriever(_includeDocumentsCommand.Context, _query.QueryParameters, _loadedDocuments, _query.IsFromStudio);
+                _timeSeriesRetriever ??= new TimeSeriesRetriever(_includeDocumentsCommand.Context, _query.QueryParameters, _loadedDocuments, _query.AddTimeSeriesNames);
```

Comment:
TimeSeriesRetriever also requires changes. The parameter there is called 'isFromStudio'

Reply:
done